Doc/library/functions.rst

```diff
@@ -541,6 +541,11 @@ are always available.  They are listed here in alphabetical order.
    :returns: The result of the evaluated expression.
    :raises: Syntax errors are reported as exceptions.
 
+   .. warning::
+
+      This function can execute arbitrary code. Calling it with
```

Comment:
Can -> is designed to?

Reply:
Good point. I will adjust that later today.

Reply:
Or just:
```suggestion
      This function executes arbitrary code. Calling it with
```

Reply:
The thing in my mind is that this is very different from ast.literal_eval, which has a warning because it's surprising for a function documented to "not be eval therefore doesn't execute arbitrary code" to have a security issue (which is unrelated to executing arbitrary code).

It's very unsurprising that an eval function executes arbitrary code, so the security warning is not something I'd personally think to add... I assume the reason is to remind people why the fact that it executes arbitrary code matters.

I think it's important to avoid accidentally giving the impression that "execute arbitrary code" is a surprising hidden behavior of eval(). It's the kind of thing that would make me afraid upon seeing such a warning that I don't understand understand the function and must surely be missing some reason why it was necessary to add that warning.

Reply:
I'Ve adapted the text like it was suggested by @JelleZijlstra.

Reply:
I still believe that this PR is currently incorrect and misleading, and it should say "is designed to" as I suggested above. :)